spec/system/user_sessions_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe "UserSessions", type: :system do
+
+  describe 'ログイン' do
+    context 'ログイン情報が正常のとき' do
+      it 'ログインに成功する' do
+      end
+      it 'フラッシュメッセージが表示' do
+      end
+    end
+    context 'ログイン情報が異常のとき' do
```

Comment:
「情報が異常」という記載だと条件の内容が曖昧なので、以下のように定義しても良いかもしれません。

上のcontextの「ログイン情報が正常のとき」の正常とうまく対比になりませんが、この書き方も好みなので正しくテストケースの内容が伝わることを優先した方が良いと思います。
```suggestion
    context 'ログイン情報が未入力のとき' do; end
    context 'ログイン情報が登録済のとき' do; end
     context 'ログイン情報のフォーマットが正しくないとき' do; end
```